simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/evse/Evse.java

```diff
@@ -250,6 +232,95 @@ public void clearToken() {
         tokenId = StringUtils.EMPTY;
     }
 
+
+    /**
+     * Try to plug connector. If EVSE is unavailable it returns false.
```

Comment:
it's questionable. Even if the Evse is unavailable, nothing prevents a user to physically plug the cable

Reply:
This is what we discussed yesterday. If EVSE unavailable - dont change the status.

Reply:
Okay, then I misunderstood.

Reply:
Removed that if-statement